build.gradle

```diff
@@ -229,8 +229,8 @@ dependencyManagement {
             entry 'spring-cloud-openfeign-core'
         }
 
-        //CVE-2022-25857
-        dependencySet(group: 'org.yaml', version: '1.31') {
+        //CVE-2022-25857, CVE-2022-38751
+        dependencySet(group: 'org.yaml', version: '1.32') {
```

Comment:
The corresponding update has been done a few hours ago. A rebase should make this change obsolete.